CHANGELOG.md

```diff
@@ -17,29 +28,27 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/) and this p
 
 ## [2.0.0] - 2017-6-20
 ### Added
-- More award has been added in award section
-- Real map has been added instead of map image in contact page
-- Form validation has been
-
-### Updated
-- Life at infratab section image layout has been updated
+- More award has been added in award section.
+- Real map has been added instead of map image in contact page.
+- Form validation has been.
```

Comment:
@priyanka1110 @moosicalbovine the sentence is not completed

Reply:
Ok, @niren777 Will update it.